pkg/registry/core/pod/strategy.go

```diff
@@ -203,6 +203,7 @@ func MatchPod(label labels.Selector, field fields.Selector) storage.SelectionPre
 	}
 }
 
+// NodeNameTriggerFunc returns value spec.nodename of given object.
 func NodeNameTriggerFunc(obj runtime.Object) string {
```

Comment:
Missed this one? Rename to `NameTriggerFunc` for consistency?

Reply:
Left this on purpose - because it's for `Pod` object, for which NameTriggerFunc would suggest it's pod`s name (as it's the case for all other triggers), whereas we are looking into `Spec.NodeName` field.

Reply:
Gotcha.